components/Wizard/formComponents/FormGroupCustom.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { FormGroup as Pf4FormGroup, TextContent, Text } from "@patternfly/react-core";
+import PropTypes from "prop-types";
+
+const showError = ({ error, touched, warning, submitError }, validateOnMount) => {
+  if ((touched || validateOnMount) && error) {
+    return { validated: "error" };
+  }
+
+  if ((touched || validateOnMount) && submitError) {
+    return { validated: "error" };
+  }
+
+  if ((touched || validateOnMount) && warning) {
+    return { validated: "warning" };
+  }
```

Comment:
I can't find `meta.warning` in the react-final-form docs, just double checking that it really exists.

https://final-form.org/docs/react-final-form/types/FieldRenderProps

Reply:
https://data-driven-forms.org/schema/validator-warnings
https://www.patternfly.org/v4/components/text-input#textinput

So the warning in meta should mimic the warning option in Patternfly's form validation

Reply:
Currently this isn't used by us but will probably be useful in the future. I.e. if a user sets a strange value that won't necessarily fail but might. Image size of 10 TB or something...